.github/workflows/cypress-test.yml

```diff
@@ -0,0 +1,27 @@
+name: Cypress End-to-End test
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
```

Comment:
I think we only need to run the tests on the incoming changes via a new pull request, so we don't need this part

Reply:
Sometimes there are direct pushes by the admins, so I guess lets keep it?

Reply:
Is that so? I don't know.

@AceTheCreator Can the bot push code directly to the main branch without first creating a branch?

Reply:
Bot can't push code directly to main branch without creating PR first.

Reply:
okay then this issue is resolved ig!